source/github-helpers/dom-formatters.ts

```diff
@@ -36,11 +35,13 @@ export function linkifyIssues(
 	// Enable native issue title fetch
 	for (const link of linkified.children as HTMLCollectionOf<HTMLAnchorElement>) {
 		const issue = link.href.split('/').pop();
-		link.setAttribute('class', 'issue-link js-issue-link tooltipped tooltipped-ne');
-		link.dataset.errorText = 'Failed to load issue title';
-		link.dataset.permissionText = 'Issue title is private';
+		link.setAttribute('class', 'issue-link js-issue-link');
+		link.dataset.errorText = 'Failed to load title';
+		link.dataset.permissionText = 'Title is private';
 		link.dataset.url = link.href;
 		link.dataset.id = `rgh-issue-${issue!}`;
+		link.dataset.hovercardType = 'issue';
```

Comment:
A small note: if `[data-hovercard-type]` is omitted (or if it also has `[data-insert-type-here-hovercards-enabled]`), it will unconditionally active.

Using `issue` is fine since it works and we can't know what type the conversation is without making a request.